internal/provider/resource_password.go

```diff
@@ -1,22 +1,93 @@
 package provider
 
 import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"golang.org/x/crypto/bcrypt"
 )
 
 func resourcePassword() *schema.Resource {
+	passwordSchema := stringSchemaV1(true)
+	passwordSchema["bcrypt_hash"] = &schema.Schema{
```

Comment:
Since this new attribute is only handled on Create, e.g. not during Read, it will also need to be handled during Import.

Reply:
Have added handling of _bcrypt_hash_ to import.